lib/kubeclient.rb

```diff
@@ -13,6 +13,9 @@ module Kubeclient
   class Client < Common::Client
     attr_reader :api_endpoint
 
+    SERVICE_ACCOUNT_TOKEN_FILE =
```

Comment:
> It's just a convenience so that users don't have to remember the service token path. It will only be used if passed in to the client constructor, not used by default.

I understand, but it still means this constant is hardcoded in the client, and I prefer to avoid maintaining it there - it means duplicating logic and chasing changes in the server side. 
Whoever uses the client can and should pass the path to the token "from the outside".